cmd/vpc/db/ping/public.go

```diff
@@ -0,0 +1,56 @@
+package ping
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/sean-/vpc/cmd/vpc/config"
+	"github.com/sean-/vpc/db"
+	"github.com/sean-/vpc/internal/command"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const cmdName = "ping"
+
+var Cmd = &command.Command{
+	Name: cmdName,
+	Cobra: &cobra.Command{
+		Use:          cmdName,
+		Short:        "ping the database to ensure connectivity",
+		SilenceUsage: true,
+		Args:         cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			log.Info().Str("command", "ping").Msg("")
+
+			var config config.Config
+			err := viper.Unmarshal(&config)
+			if err != nil {
+				log.Fatal().Err(err).Msg("unable to decode config into struct")
```

Comment:
Can you add a logger hook to catch `Fatal` log messages and close the `conswriter`?

https://github.com/rs/zerolog/blob/c3d02683c75590e4e5b5995fd74e2142d7b65426/hook.go#L9

Reply:
Let's talk about this - I'd rather remove conswriter altogether and make this a system concern instead.

Reply:
For now though, done in 4f5a621